src/main/java/io/github/prospector/orderly/config/OrderlyConfigImpl.java

```diff
@@ -64,34 +65,34 @@
     private Set<String> bosses = Sets.newHashSet(bossDefaults);
 
     static Screen createConfigScreen(Screen parent) {
-        ConfigBuilder builder = ConfigBuilder.create().setParentScreen(parent).setTitle(String.format("config.%s.title", Orderly.MODID));
+        ConfigBuilder builder = ConfigBuilder.create().setParentScreen(parent).setTitle(new LiteralText(String.format("config.%s.title", Orderly.MODID)));
```

Comment:
Why use `LiteralText`s everywhere, instead of translatable ones?

Reply:
I'm very sorry, this is my mistake.

Reply:
Are there any plans to change other configuration items to translatableText?
I see that your translation file only includes the title

Reply:
Yes, ultimately all of it will be translatable